packages/sitecore-jss-react/src/components/PlaceholderCommon.tsx

```diff
@@ -217,8 +220,15 @@ export class PlaceholderCommon<T extends PlaceholderProps> extends React.Compone
           component = this.getComponentForRendering(componentRendering);
         }
 
+        // Fallback/defaults for Sitecore Component renderings (in case not defined in component factory)
```

Comment:
Looking at this, seems like our fallback versions will always override any defined in the component factory (coming from line 220 above). Do we need to check if `!component`?

Reply:
Totally right, I added a condition statement